Docs: Zendesk dashboard template (#8955)
PostHog/posthog.com: contents/templates/zendesk-report-dashboard.mdx

---
title: Zendesk starter report template
subtitle: Help is on the way (for your support metrics)
thumbnail: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/bug_15859d0aaf.png
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/zendesk_feature_f2be443a9e.png
showTitle: true
filters:
  type:
    - dashboard
  maintainer: official
---

<Section
  divider={false}
  cols={2}
>
  <div>
    <h5>Get support metrics in PostHog</h5>
    <p>View ticket count, recent tickets, organizations and users who submit the most tickets, first reply time, and more, all in one place.</p>
  </div>
  <div>
    <h5>Combine with PostHog product data</h5>

>     <p>Combine your Zendesk data with PostHog product data to get a complete view of your user experience.</p>

Reviewer: ```suggestion
    <p>Combine your Zendesk and PostHog product data to get a complete view of your customer support experience.</p>
```

  </div>
</Section>

<section class="my-12 px-5">
  <div class="max-w-screen-2xl mx-auto">
    <h3 class="text-2xl lg:text-3xl m-0 text-center mb-6 sm:mb-12">What's on this dashboard?</h3>
    <div class="flex justify-center">
      <ul class="m-0 p-0 list-none inline-grid sm:grid-cols-2 justify-evenly relative  after:absolute after:w-full after:left-0 after:top-1/2 after:-translateY-1/2 before:absolute before:h-full before:top-0 before:left-1/2 after:-translateX-1/2  sm:after:block after:hidden sm:before:block before:hidden">
        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 ">
          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Key metric counters</h5>

>           <p class="m-0 text-[15px]">See how ticket count and average reply time are changing over time.</p>

Reviewer: ```suggestion
          <p class="m-0 text-[15px]">See how ticket counts and average reply times change over time.</p>
```
